index.js

```diff
@@ -0,0 +1,47 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] Please, consider removing the linter disables, let the linter makes its job, to find bugs in the first steps of building your project 👍